tower-sessions-core/src/session.rs

```diff
@@ -552,13 +552,16 @@ impl Session {
     /// ```
     pub fn expiry_date(&self) -> OffsetDateTime {
         let expiry = self.expiry.lock();
+        let now = OffsetDateTime::now_utc();
+
+        #[cfg(test)]
+        let now = time::macros::datetime!(1981-03-19 0:00 +0);
```

Comment:
I a better approach here is to build in some tolerance for drift. For example:

https://github.com/maxcountryman/tower-sessions/blob/bc0d0f95a56b7d71b8181281c58ecb4e881ae39e/tests/common/mod.rs#L374-L383

We could wrap this with a macro of our own to make it a little more ergonomic to use in the tests.